Makefile

```diff
@@ -9,13 +9,17 @@ NGINX_SRC_DIR ?= $(PWD)/nginx
 ARCH ?= $(shell arch)
 COVERAGE ?= OFF
 DOCKER_REPOS ?= public.ecr.aws/b1o7r7e0/nginx_musl_toolchain
+MACOS ?= false
+ifeq ($(MACOS), true)
+	PCRE2_PATH ?= /opt/homebrew/Cellar/pcre2/10.44
+	MAC_OPTIONS = -DCMAKE_C_FLAGS=-I$(PCRE2_PATH)/include/ -DCMAKE_CXX_FLAGS=-I$(PCRE2_PATH)/include/ -DCMAKE_LDFLAGS=$(PCRE2_PATH)/lib -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_C_COMPILER=clang
```

Comment:
I understand your contribution. However you can not suppose every macos user will have PCRE2 installed at this very specific location.